docs/14_0_0/validators/validateFile.md

```diff
@@ -0,0 +1,35 @@
+# validateFile
+
+`p:validateFile` is a validator which can be used to validate single and mulitple files, either of `p:fileUpload` or `h:inputFile`.
+
+## Info
+
+| Name             | Value                                         |
+|------------------|-----------------------------------------------|
+| Tag              | validateFile                                  |
+| Validator Class  | org.primefaces.validate.FileValidator         |
+| Validator Id     | primefaces.File                               |
+
+## Attributes
+
+| Name           | Default | Type    | Description     |
+|----------------| ------- |---------| ----------------- |
+| allowTypes     | null    | String  | Regular expression for accepted file types, e.g. /(\.|\/)(gif|jpe?g|png)$/
```

Comment:
worth mentionning it's a javascript regex (at it is server regex, people would expect java regex) Being said, it doesn't make a lot of sense if people use this for `h:inputFile` If I remember correctly, it's a js regex because of FileUpload widget